internal/testutil/mig/pre_check.go

```diff
@@ -0,0 +1,21 @@
+package mig
+
+import (
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func PreCheckBasic(tb testing.TB) {
+	checkLastVersion(tb)
+	acc.PreCheckBasic(tb)
+}
```

Comment:
you should consider enabling `thelper` in the linter, this and all test helpers should be 
```suggestion
func PreCheckBasic(tb testing.TB) {
	tb.Helper()
	checkLastVersion(tb)
	acc.PreCheckBasic(tb)
}
```
It will report the failing test correctly instead of flagging this function

Reply:
ohhh. right, thanks!

Reply:
@gssbzn is there a problem if tb.Helper is called multiple times?     e.g. one helper method calls another, but both can be called from tests.

Reply:
>b.Helper is called multiple times? 

No internally it flags to not consider the line but the caller the failure which will bubble up if a helper calls a helper

Reply:
thx, changed